spec/index.bs

```diff
@@ -3497,8 +3501,93 @@ The following validation rules apply:
             [=mipmap level=] |textureCopyView|.{{GPUTextureCopyView/mipLevel}}.
 </div>
 
+## Debug Markers ## {#command-encoder-debug-markers}
+
+Command encoders provide methods to apply debug labels to groups of commands or insert a single
+label into the command sequence. Debug groups can be nested to create a hierarchy of labeled
+commands. These labels MAY be passed to the native API backends for tooling, MAY be used by the user
+agent's internal tooling, or MAY be a no-op when such tooling is not available or applicable.
+
+Debug groups in a {{GPUCommandEncoder}} or {{GPUProgrammablePassEncoder}}
+must be well nested.
+
+### <dfn method for=GPUCommandEncoder>pushDebugGroup(groupLabel)</dfn> ### {#GPUCommandEncoder-pushDebugGroup}
+
+<div algorithm="GPUCommandEncoder.pushDebugGroup">
+
+    **Arguments:**
+      - {{DOMString}} |groupLabel|
+
+    **Returns:** void
+
+    Marks the beginning of a labeled group of commands for the {{GPUCommandEncoder}}.
+
+    |groupLabel| defines the label for the command group.
+
+    On the [=Content timeline=], the following steps occur:
```

Comment:
Shouldn't this be the device timeline since validation (and the state change of the encoder) are done in GPU processes?

Reply:
Sounds good! Still getting used to some of these concepts. 😃

Reply:
Done